l1-contracts/src.ts/hyperchain-upgrade.ts

```diff
@@ -443,3 +443,12 @@ async function migrateAssets(deployer: Deployer) {
   // daiTransferData;
   await deployer.executeUpgrade(deployer.addresses.Bridges.SharedBridgeProxy, 0, daiTransferData);
 }
+
+export async function transferProxyAdminErc20Bridge(deployer: Deployer){
+  const bridgeProxy: ITransparentUpgradeableProxy = ITransparentUpgradeableProxyFactory.connect(
+    deployer.addresses.Bridges.ERC20BridgeProxy,
+    deployer.deployWallet
+  );
+  const tx = bridgeProxy.changeAdmin(deployer.addresses.TransparentProxyAdmin);
```

Comment:
the admin of the proxy is the `Governance` contract, so you'll have to schedule the corresponding governance ops